tasks/CentOS.yml

```diff
@@ -11,11 +11,31 @@
   no_log: '{{ openio_repository_no_log }}'
   tags: install
 
-- name: "Configure EPEL repository"
+# For customers that refuse our EPEL internal mirror
+- name: "Configure upstream EPEL repository"
   package:
     name: "epel-release"
     state: present
-  when: openio_repository_manage_epel_repository
+  when:
+    - openio_repository_manage_epel_repository
+    - openio_repository_epel_upstream_repository
+  tags: install
+
+- name: "Configure openio EPEL mirror"
+  yum_repository:
+    name: "openio-EPEL-{{ ansible_distribution_major_version }}"
+    description: "OpenIO EPEL {{ ansible_distribution_major_version }} packages for Entreprise Linux $releasever - $basearch"
+    file: "openio-EPEL-{{ ansible_distribution_major_version }}"
+    baseurl: "{{ openio_repository_mirror_root_url }}/epel/{{ ansible_distribution_major_version }}/$basearch"
+    gpgkey: "{{ openio_repository_mirror_root_url }}/epel/{{ openio_repository_gpgkey_epel }}"
+    exclude: ['netdata']
```

Comment:
Are we going to clone epel or can we take advantage of it to remove annoying packages ?

Reply:
We have to clone exactly, customers may depend on some other packages from epel, we will already be delaying package upgrades with this...

Reply:
We will not modify this mirror.